packages/api/tests/lambdas/sf-event-sqs-to-db-records/test-write-granules.js

```diff
@@ -423,7 +501,7 @@ test('writeGranules() throws error if any granule writes fail', async (t) => {
   }));
 });
 
-test.serial('writeGranules() does not persist records to Dynamo or RDS if Dynamo write fails', async (t) => {
+test('writeGranules() does not persist records to Dynamo or RDS if Dynamo write fails', async (t) => {
```

Comment:
Why was this serial before?

Reply:
I think it was a leftover from a different test file where the tests were run in serial. These writeGranules tests moved around quite a bit so I'm thinking it just wasn't updated. I also found another one in this file:

https://github.com/nasa/cumulus/pull/2168/commits/03ec2beef864e585f45780530e82cf6419fdd250

Everything passes and I don't see a reason these would need to be run in serial.